### ISSUE TITLE
Docker build broken

### ISSUE THREAD

**Issue by dsmalko** [SUSE/Portus#2151]
Errors during docker build after upgrade to opensuse/ruby:2.6 :

```
update-alternatives: --install needs <link> <name> <path> <priority>
```

```
No provider of 'phantomjs'
```

```
ERROR:  While executing gem ... (OptionParser::InvalidOption)
    invalid option: --no-ri
        /usr/lib64/ruby/2.6.0/rubygems/command.rb:396:in `handle_options'
        /usr/lib64/ruby/2.6.0/rubygems/command.rb:307:in `invoke_with_build_args'
        /usr/lib64/ruby/2.6.0/rubygems/command_manager.rb:176:in `process_args'
        /usr/lib64/ruby/2.6.0/rubygems/command_manager.rb:146:in `run'
        /usr/lib64/ruby/2.6.0/rubygems/gem_runner.rb:59:in `run'
        /usr/bin/gem:21:in `<main>
```

**Comment by Vad1mo** [comment]
I fixed that
in https://github.com/Vad1mo/Portus/blob/master/Dockerfile#L36


```
    gem install bundler -v 1.17.3 && \
    update-alternatives --install /usr/bin/bundle bundle /usr/bin/bundle.ruby2.6 1 && \
    update-alternatives --install /usr/bin/bundler bundler /usr/bin/bundler.ruby2.6 1 && \
```

**Comment by mssola** [comment]
Fixed with #2153. Sorry for the inconvenience.
